client/app/billing/autoRenew/actions/warnPendingDebt/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "autorenew_service_debt_warn_title": "Information importante",
+  "autorenew_service_debt_warn_settle": "Régulariser",
+  "autorenew_service_debt_warn_cancel": "Annuler",
+  "autorenew_service_debt_warn_deletion": "La résiliation de ce service ne peut s’effectuer avant le paiement de la dette restante. Rendez-vous dans l’espace facturation afin de régulariser la facturation pour le service <strong>{{service}}</strong>"
```

Comment:
Do you have a link to the `espace facturation` from here ?

Reply:
It's the modal primary action